### DISCUSSION TITLE
Setup executable simply not opening.

### DISCUSSION THREAD

**Discussion by Rinrin50** [maforget/ComicRackCE#128] [Q&A] [answered]
You'll have to forgive me as I am not the most tech literate, but am just interested in compiling some manga of mine. After downloading the nightly installer from the README and opening it from my Windows Explorer, my cursor will get the blue rotating circle indicating that something is happening, but no application opens. The same happens when running the application executable from the nightly ZIP download. Attached is my device specifications.
![Screenshot 2024-12-03 220512](https://github.com/user-attachments/assets/72a3c2ea-5268-4edb-9618-b3dda1bd6a10)

Again, please forgive me if I'm missing something, but I would love some help!

**Comment by maforget** [comment, marked as answer]
You probably have the Smart Screen that pops up saying that it protected your PC. This is normal because the application isn't signed, Windows blocks it because it comes from the internet. You need to click More Info -> Run Anyway or Right Click on .exe -> Properties -> (Under General Tab) Check Unblock.

![Screenshot 2024-12-03 221427](https://github.com/user-attachments/assets/eef1784e-84b7-4598-aad5-50ce1e9258da)

**Reply by Rinrin50** [reply]
Thank you so much! This was exactly the issue.
